.env.example

```
@@ -1,3 +1,9 @@
 CONTENTFUL_SPACE_ID=jnd8s5ezvg4b
 CONTENTFUL_ENVIRONMENT=master
-CONTENTFUL_DELIVERY_TOKEN=
\ No newline at end of file
+CONTENTFUL_DELIVERY_TOKEN=
+
+GATSBY_RESHEPE_PUBLIC_KEY=replace-me
+
+# MapLibre API key
+# https://cloud.maptiler.com/account/keys/
+GATSBY_MAPLIBRE_API_KEY=replace-me
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)